Cubical/Codata/Conat/Properties.agda

```diff
@@ -0,0 +1,44 @@
+{-# OPTIONS --cubical --safe --guardedness #-}
+module Cubical.Codata.Conat.Properties where
+
+open import Agda.Builtin.Unit
```

Comment:
Why not import https://github.com/agda/cubical/blob/master/Cubical/Data/Unit.agda ?

Reply:
I didn't know that. Will fix soon